### ISSUE TITLE
Script Window: remember open files

### ISSUE THREAD

**Issue by FedeMPouzols** [mantidproject/mantid#13082]
The script window (the one that you open with F3 or View->'Script Window') has a list of recent script files, but in addition to this It would be nice if the window was able to remember the files that you have opened in (multiple) tabs when you close / start again Mantid. This can be useful sometimes, especially when you have sessions in which you are editing many files in parallel.

This has not been requested by users as far as I know, so I'm setting is a low priority, but it is actually very convenient for development.
